src/core/io/CANopen.cpp

```diff
@@ -31,6 +32,11 @@ uint8_t testerStorage[MAX_SIZE];
 
 // Queue that stores the CAN messages to send to the CANopen parser
 core::types::FixedQueue<CANOPEN_QUEUE_SIZE, core::io::CANMessage>* canQueue;
+
+// SDO variables
+void* context;
+void (*callback)(CO_CSDO* csdo, uint16_t index, uint8_t sub, uint32_t code);
```

Comment:
I left another comment about possible getting rid of these. If you do end up keeping them, you need to add more documentation to explain them